lib/actions.js

```diff
@@ -145,7 +145,10 @@ exports.type = function() {
   } else {
     this.evaluate_now(function (selector) {
       document.querySelector(selector).focus();
-    }, function() {
+    }, function(err) {
+      if(err){
+        return done(new Error(`cannot type into selector "${selector}".`));
```

Comment:
I think it might be better to have a more explicit message—“can’t find element matching selector” or something along those lines. I can see myself puzzling over this message—“well *why* couldn’t it type?”

Could maybe prefix it with the action name or add a property to the error indicating what action it came from if that’s a concern.